.github/workflows/build-website.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs'
+      - 'website'
+      - 'package.json'
+      - 'packages/**/src'
+
+  pull_request:
```

Comment:
So this builds the website when a PR is opened? Is this to validate that it _can_ build?

Reply:
Yes, we are just verifying that it can build and we have not regressed in any way in this regard. Planning to add auto-deploy on each release at least. But that will be on separate PR.

Reply:
Excellent.